.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -23,18 +23,24 @@ body:
     - macOS
     - Linux
     - Other
-- type: textarea
+- type: input
   attributes:
     label: Version of Prism Launcher
     description: The version of Prism Launcher used in the bug report.
-    placeholder: Prism Launcher 5.0
+    placeholder: Prism Launcher 5.1
   validations:
     required: true
-- type: textarea
+- type: checkboxes
+  attributes:
+    label: I've tried reproducing this issue on at least the latest stable release.
+    options:
+    - label: This bug occurs on at least the latest stable release.
```

Comment:
literally what i said above

Reply:
at least

Reply:
?

Reply:
it says "this bug occurs on AT LEAST the latest stable release".  The at least part means it can happen on the latest stable version, or any newer version (like development versions)

Reply:
what do you suggest?